.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -13,8 +13,5 @@ A clear and concise description of what the problem is. Ex. I'm always frustrate
 **Describe the solution you'd like**
```

Comment:
> Is your feature request related to a problem? Please describe.
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
> 
> Describe the solution you'd like
> A clear and concise description of what you want to happen.
> 
> Describe alternatives you've considered
> A clear and concise description of any alternative solutions or features you've considered.

Are we over-fitting this template to problems? Feels more like a bug report to me. Maybe we can make it more generic?

```
What feature are you requesting?

Are you using a workaround to do it in or outside of the product today?

How important is this feature to your continued use of the package? Can you qualify the value / importance of this feature in any way?
```

Just spitballing...